Resources/Prototypes/silicon-laws.yml

```diff
@@ -167,8 +167,21 @@
   id: IonStormLawsets
   weights:
     # its crewsimov by default dont be lame
-    Crewsimov: 0.25
+    #Crewsimov: 0.25 # Delta-V disables crewsimov as an option for borg ion storm lawsets
     Corporate: 1
     NTDefault: 1
-    Drone: 0.5
+    Drone: 0.25 # Delta-V - Lowers the spawn rate due to it being anti-crew interaction + drones are disabled anyways
     Ninja: 0.25
+    #Delta-V Lawsets Start
+    Chaplain: 1
+    Reporter: 1
+    StationEfficiency: 1
+    Cowboy: 1
+    Druid: 1
```

Comment:
missing Clown :trollface:

Reply:
i'm actually a little afraid of putting the painter & clown lawsets in as they would require modules that would require the borg to be changed; hence they'd lose the laws, and if you got the clown or painter lawset you can't really *do* your laws without inherently causing yourself to be un-ionstormed

Reply:
an engiborg with clown lawset can still tell jokes, they dont *need* the service clowning module

Reply:
you're not wrong. i've added it